style.css

```diff
@@ -1 +1,119 @@
+table {
```

Comment:
I like how you separated the CSS into multiple classes. Try to check out how BEM works, it's a naming convention for CSS. You could potentially extract common CSS into higher classes. If you're not too sure about how it works after reading about it we can work through a couple of cases together :)

Reply:
http://getbem.com/introduction/